pkgs/development/tools/skopeo/default.nix

```diff
@@ -33,8 +33,8 @@ buildGoPackage {
 
   buildFlagsArray = ''
     -ldflags=
-    -X github.com/containers/skopeo/vendor/github.com/containers/image/signature.systemDefaultPolicyPath=${defaultPolicyFile}
-    -X github.com/containers/skopeo/vendor/github.com/containers/image/internal/tmpdir.unixTempDirForBigFiles=/tmp
+    -X github.com/containers/skopeo/vendor/github.com/containers/image/v5/signature.systemDefaultPolicyPath=${defaultPolicyFile}
+    -X github.com/containers/skopeo/vendor/github.com/containers/image/v5/internal/tmpdir.unixTempDirForBigFiles=/tmp
```

Comment:
you probably do not want this override. default is /var/tmp anyway, and it's the path for big temp files according top FHS spec.

- https://github.com/Homebrew/homebrew-core/pull/45834#issuecomment-546532018